generator/generator_tests/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(
   feature_builder_test.cpp
   feature_merger_test.cpp
   filter_elements_tests.cpp
+  geo_objects_test.cpp
```

Comment:
s/test/tests/

Reply:
File name is geo_objects_test.cpp simular as feature_builder_test.cpp
Your change will cause build fail

Reply:
We use the suffix "_tests" for new files with (multiple) tests.

Reply:
I have one single test in this file, but ok, it's renamed

Reply:
What if you want to add a test later?

Reply:
I got your point and completed the notice